pkg/operation/botanist/component/kubeapiserver/configmaps.go

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubeapiserver
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/gardener/gardener/pkg/operation/botanist/component/vpnseedserver"
+	kutil "github.com/gardener/gardener/pkg/utils/kubernetes"
+	secretutils "github.com/gardener/gardener/pkg/utils/secrets"
+	versionutils "github.com/gardener/gardener/pkg/utils/version"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/apimachinery/pkg/runtime/serializer/json"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	apiserverv1alpha1 "k8s.io/apiserver/pkg/apis/apiserver/v1alpha1"
+	auditv1 "k8s.io/apiserver/pkg/apis/audit/v1"
+)
+
+const (
+	configMapAdmissionNamePrefix = "kube-apiserver-admission-config"
+	configMapAdmissionDataKey    = "admission-configuration.yaml"
+
+	configMapAuditPolicyNamePrefix = "audit-policy-config"
+	configMapAuditPolicyDataKey    = "audit-policy.yaml"
+
+	configMapEgressSelectorNamePrefix = "kube-apiserver-egress-selector-config"
+	configMapEgressSelectorDataKey    = "egress-selector-configuration.yaml"
+)
+
+var (
+	scheme *runtime.Scheme
+	codec  runtime.Codec
+)
+
+func init() {
+	scheme = runtime.NewScheme()
+	utilruntime.Must(apiserverv1alpha1.AddToScheme(scheme))
+	utilruntime.Must(auditv1.AddToScheme(scheme))
+
+	var (
+		ser = json.NewSerializerWithOptions(json.DefaultMetaFactory, scheme, scheme, json.SerializerOptions{
+			Yaml:   true,
+			Pretty: false,
+			Strict: false,
```

Comment:
I don't think so, I took this from https://github.com/gardener/gardener/blob/d5245d091bc8275986de9a53a8e3d07017682235/pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/kubelet/configcodec.go#L51. Since I'm not sure where setting it to `true` has any side-effects, I'd keep it at both places for now.